assets/package.json

```diff
@@ -11,15 +11,19 @@
     "test:watch": "vitest",
     "test:coverage": "vitest run --coverage",
     "test:ui": "vitest --ui",
-    "lint": "yarn lint:js && yarn lint:css",
+    "lint": "run-s lint:format lint:js lint:css",
+    "lint:format": "yarn prettier-check",
     "lint:js": "eslint ./src",
     "lint:css": "stylelint src/**/*.css",
-    "fix": "yarn fix:js && yarn fix:css",
+    "fix": "run-s fix:format fix:js fix:css",
+    "fix:format": "yarn prettier-write",
     "fix:js": "eslint ./src --fix",
     "fix:css": "stylelint src/**/*.css --fix",
     "e2e:start": "concurrently -kill-others --success first \"yarn start\" \"wait-on -c waitOnConfig.json -v https-get://localhost:3000 && cd e2e/ && npx cypress run\"",
     "e2e:open": "concurrently -kill-others --success first \"yarn start\" \"wait-on -c waitOnConfig.json -v https-get://localhost:3000 && cd e2e/ && npx cypress open\"",
-    "graphql:codegen": "graphql-codegen --config codegen.yml"
+    "graphql:codegen": "graphql-codegen --config codegen.yml",
+    "prettier-write": "yarn prettier --write ./src",
+    "prettier-check": "yarn prettier --check ./src"
```

Comment:
Can you move it to lint:format above?

Reply:
Perhaps we should also rename lint to check?

Reply:
Removed those. I think leaving everything as `lint` is fine and then we don't have to worry about breaking CI.